src/test/java/com/commercetools/pspadapter/paymentHandler/impl/PaymentHandlerProviderImplTest.java

```diff
@@ -147,6 +158,23 @@ public void whenPaypalPaymentIsNotApproved_transactionShouldNotBeCreated() {
         assertThat(ctpPayment.getTransactions()).isEmpty();
     }
 
+    @Test
+    public void whenOnPatchCartIsProvidedWithNoExpansion_500IsReturned() {
```

Comment:
it seems that sometimes you don't use the same naming convention for naming tests. Like at line 85.

Reply:
Yes, sometimes it feels like it's redundant to have this long name, so I just use short name. When it's not so obvious, I use the long name.